tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -1300,7 +1300,7 @@ public function testCreateIndexes(array $data): array
         ]);
 
         $this->assertEquals(400, $unknown['headers']['status-code']);
-        $this->assertEquals('Unknown attribute: Unknown', $unknown['body']['message']);
+        $this->assertStringContainsString('Unknown attribute: Unknown', $unknown['body']['message']);
```

Comment:
if we want to merge this, it would be better to have it fixed 😬

Reply:
It was already fixed in 1.6.x I just synced.